### PR TITLE
Calculate slot line

### PR DESCRIPTION
**Pull request by parkkoon**
Add naive calculate algorithm & slot logic complete
Change variable naming and state check while changing status
Calculate same symbol & slot Lines with min distance for avoiding inconsistency.
Connect winMoney & winMoneyText
Add WIN_LINE_COLOR

### CODE REVIEW

File: app/javascripts/components/playSlot/game.jsx [hunk 1]
```diff
@@ -59,54 +59,68 @@ const LINE_CASES = [
...
   [{ symbol: 3, length: 3 }, { symbol: 2, length: 4 }, { symbol: 1, length: 5 }], // 50
   [{ symbol: 2, length: 3 }, { symbol: 1, length: 4 }, { symbol: 0, length: 5 }], // 25
-  [{ symbol: 0, length: 2 }], // 10
+  [{ symbol: 0, length: 4 }], // 10
   [{ symbol: 1, length: 3 }, { symbol: 0, length: 3 }], // 5
 ];
 
+function shuffleArray(arr) {
+  for (let i = arr.length; i > 0; i -= 1) {
+    const j = Math.floor(Math.random() * i);
```
Comment (fish-sushi) [inline app/javascripts/components/playSlot/game.jsx]: This won't make random index.
I think you should use Math.floor(Math.random() * arr.length);
or you can use [Lodash's shuffle method](https://www.npmjs.com/package/lodash.shuffle).

**Review by fish-sushi — CHANGES_REQUESTED** [review]

**Comment by fish-sushi** [comment]
I'll review the rest tomorrow

File: app/javascripts/components/playSlot/game.jsx (hunk 1 above)
Comment (parkkoon) [reply app/javascripts/components/playSlot/game.jsx]: OK I will use that method.

**Review by c0smic0wl — APPROVED** [review]
🦉

**Review by fish-sushi — APPROVED** [review]
🍶

**Pull request merged**